### PR TITLE
preparing mex doc for open beta

### PR DESCRIPTION
**Pull request by WillRaphaelson**
preparing mex doc for open beta

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [x] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: docs/guides/managed-execution.md [hunk 1]
```diff
@@ -57,19 +56,16 @@ python managed-execution.py
...
 
 Note that this deployment uses flow code stored in a GitHub repository.
-If you would like to use your own flow code, push your code to your repository and specify your own repository URL and entrypoint.
 
 ### Step 3
 
-Schedule a deployment run from the UI or from the CLI.
-
-It may take a minute for the flow run to start and the logs to appear in the UI.
+Quick run the deployment from the UI or from the CLI.
```
Comment (discdiver) [inline docs/guides/managed-execution.md:62]:
```suggestion
Run the deployment from the UI or from the CLI.
```

File: docs/guides/managed-execution.md [hunk 2]
```diff
@@ -101,14 +97,7 @@ We will update this page as we make changes.
...
 
-Managed execution requires that you run one of the offered Docker images.
-You may not use your own Docker image.
-One Docker image is supported at this time: `prefecthq/prefect:2-latest`.
-We plan to support additional images with common dependencies soon.
-
-However, as noted above, you can install Python package dependencies at runtime.
-
-If you need to use your own image, we recommend using another type of work pool.
+At this time, managed execution requires that you run the official Prefect docker image: `prefecthq/prefect:2-latest`. However, as noted above, you can install Python package dependencies at runtime. If you need to use your own image, we recommend using another type of work pool.
```
Comment (discdiver) [inline docs/guides/managed-execution.md:100]:
```suggestion
At this time, managed execution requires that you run the official Prefect Docker image: `prefecthq/prefect:2-latest`. However, as noted above, you can install Python package dependencies at runtime. If you need to use your own image, we recommend using another type of work pool.
```

File: docs/concepts/work-pools.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Work pools and workers bridge the Prefect _orchestration environment_ with your
...
+Work pools organize work for execution. Work pools have types corresponding to the infrastructure that will execute the flow code, as well as the delivery method of work to that environment. Pull work pools require [workers](#worker-overview) (or less ideally, [agents](#agent-overview)) to poll the work pool for flow runs to execute. [Push work pools](/guides/deployment/push-work-pools) can submit runs directly to your serverless infrastructure providers such as Google Cloud Run, Azure Container Instances, and AWS ECS without the need for an agent or worker. [Managed work pools](/guides/deployment/push-work-pools) are administered by Prefect and handle the submission and execution of code on your behalf.
```
Comment (discdiver) [inline docs/concepts/work-pools.md:23]: Do you want this link to go to the new managed execution guide?

**Review by discdiver — COMMENTED** [review]
Nice! A few minor suggestions.

**Review by jakekaplan — APPROVED** [review]

**Pull request merged**